packages/medusa/src/types/common.ts

```diff
@@ -31,8 +31,11 @@ export type Writable<T> = {
     | FindOperator<string[]>
 }
 
-export type ExtendedFindConfig<TEntity> = FindConfig<TEntity> & {
-  where: Partial<Writable<TEntity>>
+export type ExtendedFindConfig<
+  TEntity,
+  TWhereKeys = TEntity
+> = FindConfig<TEntity> & {
+  where: Partial<Writable<TWhereKeys>>
```

Comment:
comment: May that you could integrate the [PR](https://github.com/medusajs/medusa/pull/1672/files) I mentioned above :)

Reply:
see comment above